templates/home.html

```diff
@@ -5,5 +5,5 @@
 
 <!-- wp:pattern {"slug":"twentytwentyfour/home"} /-->
 
-<!-- wp:template-part {"slug":"footer-copy"} /--></div>
```

Comment:
footer-copy and footer don't look the same, we need to address having two in a different way

Reply:
Should footer-copy be a variant footer template part?

Reply:
I think there needs to be some general refactoring of the templates so we can abstract the differences into the templates instead of into the template parts.